samclip/0.4.0/README.md

```diff
@@ -0,0 +1,27 @@
+# Samclip container
+
+Main tool: [samclip](https://github.com/tseemann/samclip)
+
+Basic information on how to use this tool:
+- executable: ./samclip
+- help: -h, --help
+- version: -V, --version
+- description: Filter SAM file for soft and hard clipped alignments
+
+Additional tools:
+
+- [samtools](https://github.com/samtools/samtools) 
+- [bwa](https://github.com/lh3/bwa)
```

Comment:
Are samtools and bwa part of the samclip executable file? I can't see where these are installed in the image.

Reply:
My bad, this was an old version of the README—I have updated it now.